testdata/builtinManifestJsonEx.jsonnet

```diff
@@ -1,22 +1,46 @@
 local a = {
-    foo: "bar",
-    bar: self.foo,
-    baz: 1,
-    bazel: 1.42,
-    boom: -1,
-    bim: false,
-    bam: true,
-    blamo: {
-        cereal: [
-            "<>& fizbuzz",
-        ],
+  foo: 'bar',
+  bar: self.foo,
+  baz: 1,
+  bazel: 1.42,
+  boom: -1,
+  bim: false,
+  bam: true,
+  blamo: {
+    cereal: [
+      '<>& fizbuzz',
+    ],
 
-        treats: [
-            {
-                name: "chocolate",
-            }
-        ],
-    }
+    treats: [
+      {
+        name: 'chocolate',
+      },
+    ],
+  },
 };
 
-std.manifestJsonEx(a, "  ")
\ No newline at end of file
+local b = [
+  'bar',
+  a.foo,
+  1,
+  1.42,
+  -1,
+  false,
+  true,
+  {
+    cereal: [
+      '<>& fizbuzz',
+    ],
+
+    treats: [
+      {
+        name: 'chocolate',
+      },
+    ],
+  },
+];
+
+{
+  a: std.manifestJsonEx(a, '  '),
+  b: std.manifestJsonEx(b, '  '),
```

Comment:
Let's also add examples of other data types, namely: null, string, number, bool. They all should work.

Reply:
sounds great :+1: